language/super_spec.rb

```diff
@@ -167,6 +167,11 @@ def a(arg)
     Super::AliasWithSuper::Trigger.foo.should == [:b, :a]
   end
 
+  it "finds super on other objects if a singleton class aliased the method" do
+    Super::SingletonAliasCase::Foo.new.alias_on_singleton
+    Super::SingletonAliasCase::Foo.new.method([]).should == [:foo, :base]
```

Comment:
@eregon - That might be a good test as well but the case that was breaking is the original method. I did just add another test that uses a singleton'ed method though